vendor/bat-native-ledger/src/bat/ledger/internal/database/database_contribution_queue.cc

```diff
@@ -170,6 +174,27 @@ bool DatabaseContributionQueue::MigrateToV23(
   return true;
 }
 
+bool DatabaseContributionQueue::MigrateToV24(
+    ledger::DBTransaction* transaction) {
+  DCHECK(transaction);
+
+  const std::string query = base::StringPrintf(
+      "ALTER TABLE %s ADD completed_at TIMESTAMP NOT NULL DEFAULT 0",
```

Comment:
Do we prefer TIMESTAMP or INTEGER for these epoch-time-in-seconds values? TIMESTAMP is really a text thing in sqlite, correct?

Reply:
@NejcZdovc Any thoughts there?

Reply:
Yeah I think we use `TIMESTAMP` just so that it's more clear compared to just `INTEGER`